lib/ae_mdw/contract.ex

```diff
@@ -306,13 +306,18 @@ defmodule AeMdw.Contract do
     end
   end
 
-  def get_init_call_details(contract_pk, tx_rec, block_hash) do
+  def get_init_call_rec(contract_pk, tx_rec, block_hash) do
```

Comment:
We should probably add typespecs to every new public function we create

Reply:
added for this and remaining functions of the module to enable credo/spec check